.github/workflows/release.crate.yml

```
@@ -0,0 +1,221 @@
+name: release crate
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  check-secret:
+    runs-on: ubuntu-latest
+    environment: crates-io-torrust
```

Let's change it to

```suggestion
    environment: crates-io-libsws
```

And the same for `publish` below.